cmake/MacroHelper.cmake

```diff
@@ -213,7 +213,7 @@ macro(nocmodl_mod_to_cpp modfile_basename)
       ${CMAKE_COMMAND} -E env "MODLUNIT=${PROJECT_BINARY_DIR}/share/nrn/lib/nrnunits.lib"
       ${NRN_NOCMODL_SANITIZER_ENVIRONMENT} $<TARGET_FILE:nocmodl>
       ${PROJECT_SOURCE_DIR}/${modfile_basename}.mod
-    COMMAND sed "s/_reg()/_reg_()/" ${PROJECT_SOURCE_DIR}/${modfile_basename}.cpp >
+    COMMAND sed "'s/_reg()/_reg_()/'" ${PROJECT_SOURCE_DIR}/${modfile_basename}.cpp >
```

Comment:
This will still break in Windows :)

Reply:
popped into my mind, and I was contemplating whether to merge it. Should have waited then! 🙄

Reply:
Follow-up PR filed, should work on at least Linii and Windowii